functions/__bobthefish_colors.fish

```diff
@@ -524,10 +524,10 @@ function __bobthefish_colors -S -a color_scheme -d 'Define colors used by bobthe
 
       set -l colorfg $base00
 
-      set -x color_initial_segment_exit     $base05 $base08 --bold
+      set -x color_initial_segment_exit     $base05 $base0B --bold  
       set -x color_initial_segment_private  $base05 $base02
-      set -x color_initial_segment_su       $base05 $base0B --bold
-      set -x color_initial_segment_jobs     $base08 $base0D --bold
+      set -x color_initial_segment_su       $base05 $base0E --bold 
+      set -x color_initial_segment_jobs     $base05 $base0C --bold 
```

Comment:
it looks like you've got trailing spaces on all your lines. mind cleaning them up?

Reply:
Yep, no problem.